pkg/utils/test/matchers/matchers.go

```diff
@@ -170,3 +171,50 @@ func expectedObjects(objs []client.Object, scheme *runtime.Scheme) map[string]cl
 
 	return objects
 }
+
+// ContainAnyOf returns a Gomega matcher that checks if the actual slice does contain at least of the wanted strings.
+func ContainAnyOf(wanted ...string) types.GomegaMatcher {
+	return &containAnyMatcher{wanted: wanted}
+}
```

Comment:
You might not need to implement a new matcher here. Instead, the `ContainAnyOf` could work with Or(), something like:

```suggestion
func ContainAnyOf(wanted ...any) types.GomegaMatcher {
	var matchers []types.GomegaMatcher
	
	for _, element := range wanted {
		matchers = append(matchers, ContainElement(element))
	}
	
	return Or(matchers...)
}
```